lib/themes/dosomething/paraneue_dosomething/scss/patterns/_counter.scss

```diff
@@ -1,12 +1,35 @@
 // @TODO: Move to Forge if this is shipped.
 .counter {
-  display: inline-block;
   color: $med-gray;
   font-weight: $weight-bold;
-  font-size: 13px !important;
-  padding-left: 3px;
+  font-size: 16px;
+  padding-left: 4px;
 
   &.is-active {
-    color: $purple;
+    color: $dark-gray;
+  }
+}
+
+.photo__actions li {
+  * {
+    display: inline;
+    vertical-align: middle;
+  }
+
+  img {
+    width: 30px !important;
+    height: 26px;
+
+    &:hover {
+      cursor: pointer;
+    }
+
+    &.is-active {
+      content: url("../images/pink-heart.svg");
+    }
+
+    &:not(.is-active) {
```

Comment:
I can take a screenshot if this doesn't make sense, but this was actually the first thing i tried. the problem was if the page loaded and the is-active class was already applied, and you unclicked the active kudo, the default gray one didn't show.